shell/platform/embedder/embedder_surface_metal.h

```diff
@@ -15,6 +17,9 @@
 
 namespace flutter {
 
+// TODO(csg): This class is Skia specific and there is another on specifically
```

Comment:
If this isn't happening imminently, then we should have an issue filed for the TODO.